Первая версия магазина

viacheslavek opened the pull request:
Первая версия магазина: нет анимаций, но есть все остальное.
-Верхнее меню (пока не закреплено)
-Карточки, изменяющие масштаб от размера экрана
-Добавлена навигация по странице
-Имеются уровни редкости

viacheslavek committed:
Создал страницу магазина:
-Верхнее меню
-Заголовки товаров с перекрестными ссылками
-Карточки
-Редкость карточек по цвету
-Создал небольшой "ассортимент"

viacheslavek committed:
Внесены правки:
-добавлен заголовок меню
-добавлены отступы по краям магазина
-улучшена сетка масштабирования
-изменено количество карточек в ширину

geogreck reviewed (changes requested):
- [x] Добавить отступы слева и справа.
- [x] Сделать меню магазина видимым на телефоне
- [x] Закрепить меню сверху при прокрутке
- [ ] Добавить предмет `mask of madness`  
- [x] Сделать ссылку на магазин в меню слева

viacheslavek committed:
Внесены правки, которые указаны Георгием в ревью
- Добавить отступы слева и справа.
- Сделать меню магазина видимым на телефоне
- Закрепить меню сверху при прокрутке

geogreck reviewed (commented):
На телефоне меню магазина перекрывает общее меню
![image](https://user-images.githubusercontent.com/74151984/179986668-69c6bb08-0ff6-4268-88df-3a82a6f5d537.png)

geogreck commented:
Принято решение понять и простить Вячеслава. Баг с меню магазина на телефоне будет перенесен в отдельное issue.
Его будет проще починить потом, так как на ветке Вячеслава стили и прочие файлы являются крайне устаревшими(ветка отстает на 31 коммит)
